src/arrayMethodSort.js

```diff
@@ -3,9 +3,34 @@
 /**
  * Implement method Sort
  */
+
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function (compareFunction = undefined) {
```

Comment:
```suggestion
  [].__proto__.sort2 = function (compareFunction = (a,b) => {......}) {
```
you need to specify the function here

Reply:
The default function was too long, so I wrote it below.